notifier/models.py

```diff
@@ -1,3 +1,28 @@
 """
-django test runner requires a models.py file.  there is nothing to see here.
+Database models for notifier.
 """
+from datetime import datetime, timedelta
+
+from django.db import models
+
+
+class ForumDigestTask(models.Model):
+    """
+    ForumDigestTask model is used for synchronization between notifier schedulers to avoid multiple
+    scheduler instances from scheduling duplicate forum digest tasks.
+    """
+    from_dt = models.DateTimeField(help_text="Beginning of time slice for which to send forum digests.")
+    to_dt = models.DateTimeField(help_text="End of time slice for which to send forum digests.")
+    node = models.CharField(max_length=255, blank=True, help_text="Name of node that scheduled the task.")
+    created = models.DateTimeField(auto_now_add=True, help_text="Time at which the task was scheduled.")
```

Comment:
Do we want a `done` field so that we can store the fact that a task was completed?